src/tests/unit/tests/electron/platform/android/rule-information.test.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { RuleInformation } from '../../../../../../electron/platform/android/rule-information';
+
+describe('RuleInformation', () => {
+    test('RuleId works correctly', () => {
+        const ruleIds = ['abc', 'xyz', 'this should work'];
```

Comment:
`ruleIds` is the same on all 3 test. The duplication can be avoided by placing the declaration right below the `describe` (and outside all the `test` functions)

Reply:
Updated as described